source/sessions/tests/README.rst

```diff
@@ -234,6 +233,39 @@ This test only applies to drivers that allow authentication to be changed on the
 * Call ``findOne`` using the session as an explicit session
 * Assert that the driver returned an error because the session is owned by a different user
 
+18. Implicit session is ignored if connection does not support sessions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This test uses a mongocryptd server as the test server; however, in the event that mongocryptd 
+starts supporting sessions or is not a viable option for this test, another server may be substituted
+in these tests as long as it does not return a non-null value for ``logicalSessionTimeoutMinutes``;
+in the event that no such server is readily available, a mock server may be used instead. 
+
+* Ensure a mongocryptd process is running
+* Create a new ``MongoClient`` pointed at the mongocryptd server with command monitoring enabled
+* Verify that the server does NOT define a value for ``logicalSessionTimeoutMinutes`` by sending a hello command to the server and checking the response
+* Send a read command to the server (e.g., ``findOne``), ignoring any errors from the server response
```

Comment:
I take it the error happens because mongocryptd doesn't actually support find or insert commands. Instead of ignoring the error can we assert that a particular error code is returned?

Reply:
Correct; and yes, we could, but in this case, we don't actually care whether the command succeeds or fails, or how it succeeds or fails, so there is no benefit to asserting on the particulars (especially since the test allows for testing against a different server) - we only care about asserting on what we send to the actual server. My preference is to keep prose tests focused on the aspects that are important to the validity of the scenario under test.

Reply:
Fair enough. Usually I worry about blindly ignoring errors but, like you point out, we're asserting on the command events so it seems fine.